src/change.md

```diff
@@ -0,0 +1,85 @@
+    so we've removed cost as a potential barrier.
+    (**Experience**)
+
+*   You'll have bought your ticket from a new company.
+    UKPA Tradiing Ltd is a brand new, wholly owned subsidiary of the UK Python Association.
+    All its profits are donated back the parent charity and we formed it in order to comply
+    fully with current charity law and best practice.
+    (**Liability**)
+
+*   There is no t-shirt, mug or any other merchandising.
+    Organising the artwork, online shop and book-keeping requires far more effort than 
+    the amount of income it generates could possibly justify and it's becoming increasingly
+    difficult to find anyone both willing and able to do so, so we've dropped it.
+    (**Burden**)
+
+*   If you are under the age of 18, you must be accompanied by a responsible adult to attend the conference.
+    There are simply too many potential issues for us to allow unaccompanied minors to attend
+    any more.
+    (**Liability**)
+
+*   There is professional looking video kit and lighting around.
+    We've reached the point where the amateur camera equipment we own is looking a little
+    long in the tooth and we don't feel it's a sensible use of our resources to replace it,
+    so we've engaged an external firm to provide that equipment for us. As they also set
+    it up and tear it down at the beginning and end of the conference, it also saves a
+    huge amount of voluntary effort.
+    (**Resources**, **Burden**)
+
+*   There are no back-to-back talk sessions.
+    We've reduced the number of talks in a single session and introduced slight breaks
+    in between them. This is to allow the 'hallway track' more chance to operate, to 
+    reduce the stress on our session chairs attempting to keep speakers to time and to
+    give a slightly less hectic feel to a day at the conference.
+    (**Experience**, **Burden**)
+
+*   Lunch timings are different.
+    It's always difficult trying to balance the needs of those who need to eat sooner 
+    than others with the desire that nobody should need to queue for lengthy periods.
+    This year, we've opted to start lunch earlier and have less time between the lunch
+    slots.
+    (**Experience**)
+
+*   Jobs Fair
+    This year, we'll use a dedicated room, provide lunch for those attending and give 
+    those sponsors who are interested in doing so, the chance to present their companies
+    and the opportunities they have available. We feel that's an improvement on the simple
+    jobs ads boards we've run in the past.
+    (**Experience**)
+
+*   Collaboration Session Introductions.
+    In the past, we've asked organisers to present their projects at the beginning of the 
+    day itself. This year, we're doing so a couple of days in advance so that anyone who
+    might not have attended previously has the chance to hear what it's all about in time
+    to make arrangements to stay.
```

Comment:
Will we do so on the day as well? So if you missed the earlier session, you can find out/learn about a great idea somebody had during the conference?